etl/command.py

```diff
@@ -180,6 +180,11 @@ def run_dag(
 
     steps = compile_steps(dag, includes, excludes, downstream=downstream, only=only)
 
+    if not steps:
+        raise ValueError(
```

Comment:
I think this should be an error rather than a warning right? The "all datasets up-to-date" case is handled below - if there are no steps here, that means invalid input